endpoints/citrus-ws/src/main/java/org/citrusframework/ws/security/SecurityHandlerFactory.java

```diff
@@ -201,13 +219,14 @@ protected void doStart() throws Exception {
      * Simple user store loads users from this factories user list.
      */
     private class SimplePropertyUserStore extends PropertyUserStore {
+
         @Override
         protected void loadUsers() {
             for (org.citrusframework.ws.security.User user : users) {
-                Credential credential = Credential.getCredential(user.getPassword());
+                Credential credential = getCredential(user.getPassword());
 
-                String[] roles = IdentityService.NO_ROLES;
-                if (user.getRoles() != null && user.getRoles().length > 0) {
+                String[] roles = new String[0];
```

Comment:
same change as in HTTP. maybe also have a look if something could be centralized.